data/json/items/armor/undergarment.json

```diff
@@ -765,7 +765,7 @@
       {
         "id": "generic_garter_belt",
         "name": { "str": "garter belt" },
-        "description": "A simple, decorated undergarment commonly used in combination with stockings.  They don't cover much, but they do help you feel sexy.",
+        "description": "",
```

Comment:
Can the description field be completely removed here? (i don't know)
If yes, then that would probably be preferable.

There's a slight difference between "empty description" and "nonexistent description" - in the former case we might accidentally do silly things like adding extra newline in order to separate the "" from the main body, which would look silly. (note, not saying that this is something we *do*, but just a class of bugs we *can* make)

```suggestion
```

Reply:
If I remember correctly descriptions only exist in either generic or variant descriptions.  So like for guns which can have variants turned off it's a requirement that they have a duplicate if the generic is still a specific weapon.  Now how does that impact item types that can't be genericized?  Absolutely no clue.